### PR TITLE
[PSDK-95] Change method names

### PR DESCRIPTION
**Pull request by yuga-cb**
### What changed? Why?
Change `list_wallet_ids` to `wallet_ids`, `save_wallet` to `save_wallet_locally!` and `load_wallets` to `load_wallets_from_local`.

The seed / wallet separation will have to wait until we support the call `user.wallets` with pagination - scoped in for v0.0.5.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,18 +144,19 @@ In order to persist the data for the Wallet, you will need to implement a store
 store(data)
 ```
 
-For convenience during testing, we provide a save_wallet method that stores the Wallet data in your local file system.
+For convenience during testing, we provide a save_wallet_locally! method that stores the Wallet data in your local file system.
 This is an insecure method of storing wallet seeds and should only be used for development purposes.
+
```
Comment (jazz-cb) [inline README.md:149]: also update faucet instructions in readme to use wallet instead of address

Comment (yuga-cb) [reply README.md:141]: Great catch. Done.

File: README.md [hunk 2]
```diff
@@ -100,21 +100,13 @@ Now, create a Wallet from the User. Wallets are created with a single default Ad
...
-
-```ruby
-# A Wallet has a default Address.
-a = w1.default_address
-a.to_s
-```
-
 Wallets do not have funds on them to start. In order to fund the Address, you will need to send funds to the Wallet you generated above. If you don't have testnet funds, get funds from a [faucet](https://docs.base.org/docs/tools/network-faucets/).
 
 For development purposes, we provide a `faucet` method to fund your address with ETH on Base Sepolia testnet. We allow one faucet claim per address in a 24 hour window.
```
Comment (jazz-cb) [inline README.md:105]: we could make this more clear
`For development purposes, we provide a `faucet` method to fund the default address in your wallet`

Comment (yuga-cb) [reply README.md:105]: Don't think we want that; the address is an implementation detail

**Pull request merged**